docs/en/docs/advanced/extending-openapi.md

```diff
@@ -248,3 +248,27 @@ Now, to be able to test that everything works, create a *path operation*:
 Now, you should be able to disconnect your WiFi, go to your docs at <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>, and reload the page.
 
 And even without Internet, you would be able to see the docs for your API and interact with it.
+
+
+### Add Tags
+
+You can group path by tags in both docs and redoc by using tags, tags also allows you to put descriptions and links. Tags description suppports markdown.
+
+```Python hl_lines="9-22 56"
+{!../../../docs_src/extending_openapi/tutorial003.py!}
+```
+
+### Change the *path operation* order
+
+It is good to have a documentation, but a sorted one is better, you can use tags_sorter and operations_sorter to sort your docs.
+See this thread for the underlying logic (Swagger UI 3.17.6 required, Redoc not affected) :
+<a href="https://github.com/swagger-api/swagger-ui/issues/2990" class="external-link" target="_blank">https://github.com/swagger-api/swagger-ui/issues/2990</a>
```

Comment:
Is there a doc where all of them are listed? Dislike the fact it's linking to an issue, couldn't figure out the complete list myself.

Reply:
Considering it's not in a separate table I doubt you could make a good highlight, think it's still better link than github issue, just need a way to point to the correct variables

Reply:
I included the link to this general swagger doc and put as a technical note the link to the issue describing the technical implementation